share/gpodder/extensions/enqueue_in_mediaplayer.py

```diff
@@ -158,37 +152,36 @@ def on_error(exception):
             try:
                 import pathlib
                 self.url = pathlib.Path(filename).as_uri()
-            except:
+            except ImportError:
                 self.url = 'file://' + filename
-            logger.debug('URL: %s', self.url)
-        self.position = pos * 1000 * 1000  # pos in microseconds
+        self.position_us = pos * 1000 * 1000  # pos in microseconds
         if self.init_dbus():
             # async to not freeze the ui waiting for the application to answer
             self.player.OpenUri(self.url,
                                 dbus_interface=self.INTERFACE_PLAYER,
                                 reply_handler=on_reply,
                                 error_handler=on_error)
         else:
-            self.wait_for_player(filename, pos)
+            self.prepare_wait_for_player(filename, pos)
```

Comment:
I only looked at this long enough to confirm you made the changes thp requested, and this is only an observation and not anything I am saying needs to be changed.  The name `prepare_wait_for_player` makes me expect somewhere there will be a "wait for player" and it is not immediately obvious where this happens.  I'm guessing `open_files` has that side-effect.